scripts/prepare_datasets/prepare_datasets.py

```diff
@@ -77,8 +74,30 @@
     lines.append(f'| {sortid} | [view]({le_url}) | {description} |')
 lines.append('')
 
+lines.append('')
+lines.append(f'[Browse all recordings]({aws_url}/default?feed={x.get_uri()})')
+
 txt = '\n'.join(lines)
 print('~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~')
 print(txt)
 print('~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~')
 
+try:
+    with open('README.md') as f:
+        readme_txt = f.read()
+    ind1 = readme_txt.find('<!-- BEGIN DATA TABLE -->')
+    ind2 = readme_txt.find('<!-- END DATA TABLE -->')
+    assert (ind1 >=0) and (ind2 >= 0)
+    readme_txt = ''.join([
+        readme_txt[:ind1],
+        '<!-- BEGIN DATA TABLE -->\n',
+        f'\n<!--- Auto-generated at {datetime.datetime.now().strftime("%m/%d/%Y, %H:%M:%S")}-->\n',
+        '\n'.join(lines),
+        '\n',
+        readme_txt[ind2:]
+    ])
+    with open('README.md', 'w') as f:
+        f.write(readme_txt)
```

Comment:
I suppose we have this in version control, so there's no real concern about checking & keeping a backup copy first.

Reply:
y